addons/survey/static/tests/components/question_page_one2many_field_tests.js

```diff
@@ -301,4 +304,60 @@ QUnit.module("QuestionPageOneToManyField", (hooks) => {
             assert.strictEqual(target.querySelector(".o_is_section [name=title]").innerText, "a");
         }
     );
+
+    QUnit.test("full page list question creation", async (assert) => {
+        const questions = [];
+        for (let i = 1; i <= 45; i++) {
+            questions.push({
+                id: i,
+                is_page: false,
+                title: `question ${i}`,
+                sequence: i,
+            });
+        }
+
+        serverData.models.survey_question.records = questions;
+        serverData.models.survey.records[0].question_and_page_ids = questions.map((q) => q.id);
+        await makeView({
+            type: "form",
+            resModel: "survey",
+            resId: 1,
+            serverData,
+            arch: `
+            <form>
+                <field name="question_and_page_ids" widget="question_page_one2many">
+                    <tree>
+                        <field name="is_page" invisible="1" />
+                        <field name="title" />
+                        <field name="random_questions_count" />
+                        <field name="sequence" widget="handle"/>
```

Comment:
With the fixup enabled (and the override of onAdd removed), you need to add this for the test to be successful:
```xml
<control>
    <create name="add_question_control" string="Add a question" context="{'question_create': True}"/>
</control>
```